tests/gp/test_hsgp_approx.py

```diff
@@ -194,17 +197,20 @@ def test_conditional(self, model, cov_func, X1, parameterization):
         prior and compare them using MMD two sample test.  Tests both centered and non-centered
         parameterizations.  The conditional should match the prior when no data is observed.
         """
+        seeds = np.arange(5) + 197  # 5 possible seeds
+        rng = np.random.default_rng(np.random.choice(seeds))
+
```

Comment:
It seems cleaner to use `@parameterize` like for `test_prior`?

Reply:
The @parametrize was just to show what seed caused the test to fail. The idea was to revert back to choosing one of 5 possible seeds once the root cause was found.

I imagine we don't want to run each test 5 times, but instead allow some variation across tests.